addons/product/models/product_combo_item.py

```diff
@@ -26,6 +26,11 @@ class ProductComboItem(models.Model):
         related='product_id.lst_price',
     )
     extra_price = fields.Float(string="Extra Price", digits='Product Price', default=0.0)
+    active = fields.Boolean(
```

Comment:
Hi @dere-odoo 
You can't add a (stored) field in stable, so I'm afraid this solution won't work here.